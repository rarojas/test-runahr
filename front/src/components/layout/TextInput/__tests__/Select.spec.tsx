import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import { Select } from '../index';

const defaultProps = {
    input: { value: '1' },
    label: 'label',
    type: 'text',
    meta: { touched: false, error: false },
    children: (
      <>
        <option value="1">1</option>
      </>
    )
  },
  TestSelect = (props: any) => <Select {...defaultProps} {...props} />;

describe('Render DateInput', () => {
  it('render correctly date component', () => {
    const SelectInputComponent = renderer.create(<TestSelect />).toJSON();
    expect(SelectInputComponent).toMatchSnapshot();
  });
});
