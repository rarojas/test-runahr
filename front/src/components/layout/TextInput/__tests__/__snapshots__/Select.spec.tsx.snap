// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render DateInput render correctly date component 1`] = `
Array [
  <small
    className="sc-jWBwVP jvELBg"
  >
    label
  </small>,
  <select
    className="sc-kkGfuU ciotAl"
    placeholder="label"
    value="1"
  >
    <option
      value="1"
    >
      1
    </option>
  </select>,
]
`;
